### PR TITLE
Compile for tvOS

### PR DESCRIPTION
**Pull request by Hengyu**
This PR enables the package to be compiled for tvOS. While this PR does not guarantee the performance (UI responsibility) on tvOS.

### CODE REVIEW

**Comment by Hengyu** [comment]
tvOS has its own user interface preference, we can use this package's core purchase flow and having our own UI optimize for tvOS.

File: Sources/StoreHelper/Styles/Shared/Buttons.swift [hunk 1]
```diff
@@ -7,19 +7,36 @@
 
 import SwiftUI
 
-@available(iOS 15.0, macOS 12.0, *)
 public extension View {
-    func xPlatformButtonStyleBorderless() -> some View { self.buttonStyle(.borderless) }
```
Comment (Hengyu) [inline Sources/StoreHelper/Styles/Shared/Buttons.swift:12]: no functional changes, update to compile on tvOS

File: Sources/StoreHelper/Styles/Shared/TapGesture.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import SwiftUI
+
+extension View {
+
+    @ViewBuilder func xPlatformOnTapGesture(perform: @escaping () -> Void) -> some View {
+        #if os(tvOS)
+        if #available(tvOS 16.0, *) {
+            onTapGesture(perform: perform)
+        } else {
+            onLongPressGesture(minimumDuration: 0.2, perform: perform)
```
Comment (Hengyu) [inline Sources/StoreHelper/Styles/Shared/TapGesture.swift:17]: on tvOS 15, use long press gesture as a fallback due to the tap gesture is available starting tvOS 16

**Review by russell-archer — APPROVED** [review]
Great work! Thanks Hengyu!

**Pull request merged**